start/routes.ts

```diff
@@ -9,15 +9,30 @@
 
 import env from '#start/env'
 import router from '@adonisjs/core/services/router'
+
 const FilesController = () => import('#controllers/files_controller')
+const BuildingsController = () => import('#controllers/buildings_controller')
 
 router.get('/', async () => {
   return { appName: env.get('APP_NAME'), version: env.get('APP_VERSION') }
 })
 
 router
   .group(() => {
-    router.get('files/:key', [FilesController, 'get'])
-    router.post('files', [FilesController, 'post'])
+    router.get('/:key', [FilesController, 'get'])
+    router.post('/', [FilesController, 'post'])
+  })
+  .prefix('api/v1/files')
+
+router
+  .group(() => {
+    // fetches campuses specifically
+    router.get('/campuses/:id', [BuildingsController, 'getByCampus'])
+    router.get('/campuses', [BuildingsController, 'getCampuses'])
+
+    // fetches buildings specifically
+    router.get('in-campus/:id', [BuildingsController, 'getByCampus'])
+    router.get('/:id', [BuildingsController, 'getById'])
+    router.get('/', [BuildingsController, 'getAll'])
   })
-  .prefix('api/v1')
+  .prefix('api/v1/buildings')
```

Comment:
1. `/api/v1/buildings/in-campus/:id` is a duplicate of `/api/v1/buildings/campuses/:id`
2. `/api/v1/buildings/campuses` doesn't seem like the most logical path to get a listing of campuses. Maybe this should be exposed outside of the buildings group? (`/api/v1/campuses`)
   Or maybe it would make more sense to start with campuses? `/api/v1/campuses` for a list of campuses, `/api/v1/campuses/buildings` for a list of all buildings, `/api/v1/campuses/:id/buildings` for a list of buildings in a given campus?